src/components/common/Footer.jsx

```diff
@@ -0,0 +1,81 @@
+                      <ul className="text-gray-600 dark:text-gray-400">
+                          <li className="mb-4">
+                              <a href="https://flowbite.com" className="hover:underline">Flowbite</a>
+                          </li>
+                          <li>
+                              <a href="https://tailwindcss.com/" className="hover:underline">Tailwind CSS</a>
+                          </li>
+                      </ul>
+                  </div> */}
+                  <div className="text-center sm:text-start mb-6 sm-mb-0">
+                      <h2 className="mb-2 sm:mb-3 text-sm font-semibold text-gray-900 uppercase dark:text-white">Navigation</h2>
+                      <ul className="text-gray-600 dark:text-gray-400">
+                          <li className="mb-1">
+                              <a href="/" className={`${pathName === "/" && "text-blue-700 font-bold" } hover:underline`}>Home</a>
+                          </li>
+                          <li className="mb-1">
+                              <a href="/about" className={`${pathName === "/about/" && "text-blue-700 font-bold" } hover:underline`}>About Us</a>
+                          </li>
+                          <li>
+                              <a href="/contact" className={`${pathName === "/contact/" && "text-blue-700 font-bold" } hover:underline`}>Contact Us</a>
+                          </li>
+                      </ul>
+                  </div>
+                  <div>
+                      {/* <h2 className="mb-6 text-sm font-semibold text-gray-900 uppercase dark:text-white flex whitespace-nowrap"><img className="size-6 mr-1" src="/images/address.png" alt="Phone"/> Address ABC - XYZ</h2> */}
+                      <ul className="text-gray-600 dark:text-gray-400">
+                          <li className="mb-4">
+                              <a href="#" className="hover:underline flex"><img className="size-6 mr-2" src={addressIcon} alt="Phone"/> Abcde fghijkl mnop qrstuvw xyz</a>
+                          </li>
+                          <li className="mb-4">
+                              <a href="#" className="hover:underline flex"><img className="size-6 mr-2" src={phoneIcon} alt="Phone"/> +910123456789</a>
+                          </li>
+                          <li>
+                              <a href="#" className="hover:underline flex"><img className="size-6 mr-2" src={emailIcon} alt="Phone"/> hindflo@gmail.com</a>
+                          </li>
+                      </ul>
+                  </div>
+              </div>
+          </div>
+          <hr className="my-6 border-gray-200 sm:mx-auto dark:border-gray-700 lg:my-8" />
+          <div className="sm:flex sm:items-center sm:justify-between">
+              <span className="text-sm text-gray-500 sm:text-center dark:text-gray-400">© 2024 <a href="#" className="hover:underline">Hindflo™</a>. All Rights Reserved.</span>
```

Comment:
You should use year from date instead of hardcoded year.
And maintain a constant for `Hindflo` and use that constant across, so that in future if you have to change the name of the brand, it will be just one change.